src/azure-cli/azure/cli/command_modules/mysql/tests/latest/test_mysql_scenario.py

```diff
@@ -209,6 +214,35 @@ def _test_flexible_server_mgmt(self, database_engine, resource_group):
 
         self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(database_engine, resource_group, restore_server_name), checks=NoneCheck())
 
+    def _test_mysql_flexible_server_import_create_mgmt(self, database_engine, resource_group):
+        storage_size = 32
+        version = '5.7'
+        location = 'eastus'
+        sku_name = 'Standard_B1ms'
+        tier = 'Burstable'
+        resource_group = 'nitishsharma-group'
```

Comment:
I strongly recommend that you can leverage resource group from function entry. It will be generated random resource group name. Because these cases not only for unit test but also used to end to end testing in pipeline.

Reply:
And Can we update the location to reference DEFAULT_LOCATION = "eastus2euap"? If this location cannot support your feature, It's OK for now.

Reply:
I have removed my own rg from the test and I am using the rg from the function entry now. I tried using DEFAULT_LOCATION for test but I faced problems there in creation of single server so using westus2 insted.